specification/components/r4/examples/R4-OperationOutcome.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Just for traceability, as stated in the STU3 PR, we don't commit these examples to git. During `make publish` they copied over to the sandbox, so the directory `specification/components/r4/examples/` should be added to .gitignore